server/neptune/workflows/internal/activities/github.go

```diff
@@ -48,6 +54,11 @@ type UpdateCheckRunResponse struct {
 }
 
 func (a *githubActivities) UpdateCheckRun(ctx context.Context, request UpdateCheckRunRequest) (UpdateCheckRunResponse, error) {
+	// TODO: Use this jobURL and populate the summary with the link to the log streaming UI
+	_, err := a.JobURLGenerator.GenerateJobURL(request.JobID)
```

Comment:
I don't think you took a look at my PR at all, but you're trying to do all the things I did
https://github.com/lyft/atlantis/commit/f0398331ed64ed3f3c0418f21b1c664fd6281a67